src/controllers/accommodation.js

```diff
@@ -1,7 +1,7 @@
 import { config } from 'dotenv';
 import _ from 'lodash';
 import models, { Accommodation, Bookings } from '../db/models';
-import Accomodation from '../helpers/queries';
+import query from '../helpers/queries';
```

Comment:
Why did you rename `Accomodation` to `query`?
I think `Accommodation` is a good name to give to a class of accommodations related queries.

Reply:
It has the same name as our table name

Reply:
Having the same name as table does it affect anything?